### ISSUE TITLE
Komodo IDE 10 - Cannot close files

### ISSUE THREAD

**Issue by SecHobo** [Komodo/KomodoEdit#1496]
### Short Summary
Files cannot be closed via

clicking x on the tab (but switching open files on the tab works)
apple-W
file->close (it is grayed out for all open files)

you can open files but nothing seems to be able to close them
 
### Steps to Reproduce

Komodo IDE, version 10.0.0, build 89159, platform macosx.
Built on Wed May 11 15:42:24 2016.

happens with the version downloaded directly from the site and the one from my account.  the license is installed and activated correctly.

### Expected results

I'd like to close files via all the supported methods

### Actual results

frustration at not being able to close files

### Platform Information

*Komodo Edit or IDE?* IDE
*Komodo Version?* 
*Operating System (and version)?* 

Komodo IDE, version 10.0.0, build 89159, platform macosx.
Built on Wed May 11 15:42:24 2016.

### Additional Information

Keep using Komodo 9 because 10 is useless in this state

**Comment by Naatan** [comment]
It's working fine for me. Could you please share your error log after attempting to close the file? Help > Troubleshooting > View Log File.

**Comment by SecHobo** [comment]
It was an addon thing, restart without addons and it works. I saw lots of 

[2016-05-17 12:33:44,278] [ERROR] console-logger: SyntaxError: expected expression, got '<<' (2) in chrome://html-tools/content/js/main.js:129
Traceback (most recent call last):
  File "chrome://html-tools/content/js/main.js", line 129, in


in the logs (thanks by the way I didn't even know that existed

when activating the old add on manager via 

ko.windowManager.openOrFocusDialog("chrome://mozapps/content/extensions/extensions.xul", "Addons:Manager")

(I couldn't figure out how to delete an addon from commando (documentation??) I get a warning "Add-on compatibility checking is disabled..."  Don't remember doing that ...

Re-enabling that killed a bunch of add-ons ant it is working fine now

I believe that was the core issue - incompatible addons inherited from my Komodo IDE 9 install and not automatically disabled by 10.  Funny thing, even after enabling it and removing all incompatible addons when I restart Komodo 10 addon compatibility checking is disabled again.

**Comment by Naatan** [comment]
Thanks for the follow up @SecHobo, I'll remove the addon from our packages and notify the author.

**Comment by Naatan** [comment]
Did you use the pre-release btw? Komodo 10 should have removed any incompatible addons, unless the 10.0 profile already existed (which would be the case if you used the pre-release).

**Comment by SecHobo** [comment]
No, this is the first build of 10 I've tried.  I got an email from ActiveState MarCom whose subject was "Your Komodo IDE X Upgrade Is Available" so I logged in and grabbed that version.  I have OSX config files for 8.5, 9.1, 9.2, 9.3 and now 10.0 (which was created today) in my ~/Library/Application Support/KomodoIDE/ directory.  I think the problem was that the automatic checker was off and does not seem to want to stay on.

**Comment by Naatan** [comment]
Hmm that's odd, I'll have another look and make sure that the addon uninstaller is working. Thank you for the info.

**Comment by manngo** [comment]
I’m the author of HTML Tools, and I’ve just discovered this thread. I’d appreciate more information on the incompatibility of the addon so I can fix it.

I was having trouble getting KE10 working at all until I renamed the KomodoEdit folder in my Application Support folder (I’m using OSX) and forced it to create a new profile without migrating anything. I then installed a fresh copy of the addon, telling it to install it despite the version incompatibility.

The error message cited above includes `chrome://html-tools/content/js/main.js` which certainly doesn’t appear to contain problematic code. This leads me to suspect that the version installed above is out of date. (There was one aberrant version which was subject to errors of synchronisation, but I thought I replaced it before anybody fell prey to it).

I have also discovered some other issues:

- following includsion: `<script type="application/x-javascript" src="chrome://xtk/content/xtk.js" />` appears to break things
- `window.sizeToContent()` seems to break things.

I have attached a current version of the addon with these problems disabled. I would appreciate feedback before I finish any remaining issues.

Thanks

[html-tools-1.2-ko.xpi.zip](https://github.com/Komodo/KomodoEdit/files/289533/html-tools-1.2-ko.xpi.zip)
